front-end/src/pages/calendar/CalendarComponent.js

```diff
@@ -107,6 +112,20 @@ export class CalendarComponent extends BaseComponent {
     });
   }
 
+   // Update calendar UI to underline dates with entries
+   updateCalendar() {
+    const dateCells = document.querySelectorAll(".day"); // Assuming these are your date cells
+
+    dateCells.forEach((cell) => {
+      const cellDate = cell.getAttribute("data-date"); // Assuming each day cell has a data-date attribute
+      if (this.datesWithEntries.includes(cellDate)) {
+        cell.classList.add("underline"); // Add the underline class to the cell
```

Comment:
I suggest calling the class something that is clear what separates this element from others, such as "logged-data" or "recorded-data"

Reply:
Only because we might want to change it from an underline to a color or an emoji later down the line

Reply:
The reason why I had to resort to an underline is due to the lack of feature for logging in for previous day. Once the navigation to change the days in the check in component is implemented, we can change the underline feature. What do you think?